resolvers.ts

```diff
@@ -263,6 +264,19 @@ export const resolvers: Resolvers = {
       }
       return return_obj;
     },
+    ZipLink: async (root, args, context, info) => {
+      const res = await getFullResponse(`/workflow_runs/${args.workflowRunId}/zip_link.json`, args, context)
+      if (res.status !== 302) {
+        return {
+          url: null,
+          error: res.statusText,
+        };
+      }
```

Comment:
So even if the response is 200 you want to deliver `null` as the url field?

Reply:
Yes, in this case. A 200 will actually represent an error given how the rails endpoint is set up. It returns a 302 with the url in the header.